### PR TITLE
[FEATURE] Change theme color

### LINKED ISSUE #818: [FEATURE] Change theme
### Description

I would like to change app theme color .

### Screenshots

Screenshot 2023-07-10 155832

### Additional information

_No response_

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I have checked the existing issues
- [X] I'm a GSSoC'23 contributor
- [X] I want to work on this issue

### PR DESCRIPTION
**Pull request by BhartiNagpure**
#818 Change theme color

close #818 

In this I am change the whole theme color of app .
also solve minor bug that is soundsetting route and apply active tab CSS in setting page.

I think you loved this theme.
Screenshot 2023-07-16 122528

Screenshot 2023-07-16 122512

### CODE REVIEW

**Comment by BhartiNagpure** [comment]
@narayan954 I'm done with my work please review it.

**Comment by narayan954** [comment]
> 

Thank you! Reviewing!

**Review by narayan954 — COMMENTED** [review]
little syntax error,fixing that and checking

File: src/components/Post/index.jsx [hunk 1]
```diff
@@ -174,6 +174,13 @@ function Post(prop) {
   };
 
   return (
+    <div
```
[the remark this answers is not in the document]
Comment (BhartiNagpure) [reply src/components/Post/index.jsx:177]: > this div wasnt closed,fixing that

@narayan954 that div close in the last.

**Comment by narayan954** [comment]
> @narayan954 I'm done with my work please review it.

I think there was a merge conflict and while solving the merge conflict you missed some syntax, hence there's an syntax error in the app, I'll be reverting the merge commit to fix it cleanly! :)

**Comment by BhartiNagpure** [comment]
@narayan954 check now is there any issue.

**Comment by narayan954** [comment]
> @narayan954 check now is there any issue.

i was resolving/merging it :/ anyways, checking again

**Comment by narayan954** [comment]
let me resolve this one

**Review by narayan954 — APPROVED** [review]
looks good to me!

**Comment by narayan954** [comment]
Thanks for your contribution! :)

**Pull request merged**